src/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs

```diff
@@ -243,15 +254,23 @@ public void RootId()
         public void StartStopWithTimestamp()
         {
             var activity = new Activity("activity");
-            Assert.Throws<InvalidOperationException>(() => activity.SetStartTime(DateTime.Now));
+            Assert.Equal(activity.StartTimeUtc, default(DateTime));
```

Comment:
Nit: the arguments to Equal are expected and actual, so these should be swapped (it's relevant if the test fails, because the exception message that's generated calls them out by name)